packages/moxb/src/progress/Progress.ts

```diff
@@ -0,0 +1,7 @@
+import { Value } from '../value/Value';
+
+export interface Progress extends Value<number> {
+    status?: 'success' | 'exception' | 'normal' | 'active';
```

Comment:
It would be nice to add some code comments here, explaining what the different states. mean. (I can figure our `success` and `exception`, but `active` and `normal` are not self-explanatory...)

Reply:
Added description